demo/pymediasoup-client/src/mediasoup.py

```diff
@@ -468,7 +496,7 @@ async def close(self):
 
     # Use wsurl if provided. Otherwise construct URI from default dev demo server
     if not args.wsurl:
-        uri = f"wss://dts-webrtc.dev.2060.io:443/?roomId={args.room}&peerId={peerId}"
+        uri = f"wss://webrtc.dev.2060.io:443/?roomId={args.room}&peerId={peerId}"
```

Comment:
I think there is no need to change this default value, since the pymediasoup demo is used by DTS that use the dts-webrtc deployment

Reply:
Solved !